src/components/interaction-designer/block-editors/BlockId.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="text-right">
+  <div class="block-id text-right">
```

Comment:
Note: I took the opportunity to add  missing `class` on existing component to match our convention https://docs.google.com/document/d/1TDnCLixJKGAEIcW2V8prlHDIqrh8UVh-d0bwXOIXX3Q/edit#

Reply:
Thank you! 👌